docs/project_ram/RAM_1pager.md

```diff
@@ -0,0 +1,48 @@
+
+## What makes for an impactful RAM engagement?
+In our RAM impact matrix, we consider multiple dimensions of a research project and team to prioritise engagement across Turing's research projects:
+* **Applicability**: How much potential does a research project have for real-world use?
+* **Capability**: To what extent does the team have the skills to translate their research to real-world use?
+* **Values**: How much does the team's approach to research align with our focus areas (e.g. interoperable open source tools, interdisciplinary collaboration, participatory co-creation)
+* **Practices**: How ready is the team to adopt RAM ways of working?
+ 
+![RAM rhombus-Simple diagram drawio](https://github.com/alan-turing-institute/research-application-management/assets/5104098/19d17aff-5caa-441f-8703-1497553b1ab3)
+
+## What do RAMs bring to research teams?
+
+Effective research is about more than simply answering a specific research question. For a project to maximise its chance for impact the team must also consider questions like:
+- Do I know who may use this research when it is finished, and how they may benefit from it?
+- Do I know everyone who will be affected by the research, and what they want to see from it?
+- Have I set up my research project to outlast any specific funding window?
+- Do I know how my work fits into the wider landscape of research in my field?
+
+The RAM skillset is focused on effectively exploring and answering these questions. Below are some examples of what this could entail:
+
+| Focus area | Rationale | Example activities |
+| ---------- | --------- | ------------------ |
+| End-to-end engagement and co-creation | Ensuring that all impacted and involved people, particularly end-users, are identified early, brought into the research project and actively considered when shaping outputs | Stakeholder mapping; stakeholder engagement pipelines; organising; facilitating and managing engagement activities; translating engagement activities into research questions |
+| Usability and accessiblity of research outputs | Ensuring anybody who wants to engage in the research outputs can | Writing cleaner, easier to understand code; creating documentation, guides and educational resources to support core research outputs; open-sourcing research (where possible) |
+| Effective project strategy | Prioritising work to practically realise as much potential as possible from limited resources, funds and time | Creating a team strategy and roadmap; identifying sustainability options after funding windows end; identifying and implementing a critical path to success |
+
+Most importantly, a key quality of RAMs is flexibility, as there is no one-size-fits-all approach to delivering impact for research project teams. A big part of the RAM role is identifying the most suitable pathway to impact for any particular project team, and leading on the implementation of that pathway.
```

Comment:
```suggestion
Most importantly, a key quality of RAMs is flexibility, as there is no one-size-fits-all approach to delivering impact for research project teams. Some activities above will be useful to project teams, some won't, and some are as of yet undefined. A big part of the RAM role is identifying the most suitable pathway to impact for any particular project team, and leading on the implementation of that pathway.
```

Reply:
This is from a convo with @f-rower... I think we should have some explicit wording saying 'we don't know what the answer to impact for ur research project is, but we're here to try and figure it out'. Adding it as a suggestion so you can see the difference (couldn't quite convince it def should be added). Happy to have a discussion about it!